docs/README.md

```diff
@@ -20,56 +20,58 @@ Thank you!
 
 ### Test credentials
 
-Please note that not all payment methods support testing. The payment methods that support testing payments are enabled.
+Please note that not all payment methods support testing, so only the payment methods that support testing payments are enabled for these credentials.
 
 * Merchant Id: `375917`
```

Comment:
Besides with the saippuakauppias, I can now use the api with merchant in test mode, right-o?